opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationResult.java

```diff
@@ -305,8 +307,8 @@ public AccountInformationResult user_anton_brueckner_provided_initial_parameters
 
         ExtractableResponse<Response> response = RestAssured
                 .given()
+                    .header(X_REQUEST_ID, DEFAULT_X_REQUEST_ID)
```

Comment:
Consent API calls should use generated request IDs (like it was before)

Reply:
random UUID is generated